test/cookbooks/macos_test/recipes/preferences.rb

```diff
@@ -2,16 +2,40 @@
   path '/Users/vagrant/Library/Preferences/com.apple.finder.plist'
   entry 'AppleShowAllFiles'
   value true
+  owner 'vagrant'
+  group 'staff'
 end
 
 plist 'put the Dock on the left side' do
   path '/Users/vagrant/Library/Preferences/com.apple.dock.plist'
```

Comment:
You can create a variable for `/Users/vagrant/Library/Preferences/com.apple.dock.plist` to avoid some of the code repetition